packages/@react-aria/tag/src/TagKeyboardDelegate.ts

```diff
@@ -86,6 +96,50 @@ export class TagKeyboardDelegate<T> extends GridKeyboardDelegate<T, GridCollecti
       if (this.focusMode === 'row') {
         return key;
       }
+    } else {
+      return this.getLastKey();
+    }
+  }
+
+  getKeyPageAbove(key) {
+    let item = this.collection.getItem(key);
+    if (!item) {
+      return;
+    }
+
+    // If focus is on a cell, start searching from the parent row
+    let newKey;
+    if (this.isCell(item)) {
+      newKey = item.parentKey;
+    }
+
+    // Find the previous item
+    newKey = this.findPreviousKey(newKey);
+
+    // PageUp if there's something to page up to
+    if (newKey) {
+      return this.getKeyAbove(key);
+    }
+  }
```

Comment:
```suggestion
  getKeyPageAbove(key) {
    return this.getKeyAbove(key);
  }
```

Reply:
I originally had that, but decided to change it since getKeyAbove will wrap to first/last key when it reaches the end/beginning and wasn't sure if we want the behavior with PageUp/Down

Reply:
Ah, I see, I think wrapping is the right thing to do, page up and page down appear to act like the arrow keys
https://spectrum.adobe.com/page/tag/#Keyboard-interactions
at least, that's how I'm interpreting last becomes first, is that it wraps